[E 250306 11:10:24 smartConnect:234] Error occurred while making a GET request to https://apiconnect.angelone.in/rest/secure/angelbroking/portfolio/v1/getAllHolding. Headers: {'Content-type': 'application/json', 'X-ClientLocalIP': '127.0.0.1', 'X-ClientPublicIP': '106.193.147.98', 'X-MACAddress': '6a:e0:7a:cc:09:b3', 'Accept': 'application/json', 'X-PrivateKey': '8mCgbjiE', 'X-UserType': 'USER', 'X-SourceID': 'WEB', 'Authorization': 'Bearer eyJhbGciOiJIUzUxMiJ9.eyJ1c2VybmFtZSI6IlM1NDg2NDIxNCIsInJvbGVzIjowLCJ1c2VydHlwZSI6IlVTRVIiLCJ0b2tlbiI6ImV5SmhiR2NpT2lKU1V6STFOaUlzSW5SNWNDSTZJa3BYVkNKOS5leUoxYzJWeVgzUjVjR1VpT2lKamJHbGxiblFpTENKMGIydGxibDkwZVhCbElqb2lkSEpoWkdWZllXTmpaWE56WDNSdmEyVnVJaXdpWjIxZmFXUWlPamdzSW5OdmRYSmpaU0k2SWpNaUxDSmtaWFpwWTJWZmFXUWlPaUkxWkdVM1lqazROUzB3WVRrd0xUTmtNREl0T1RZMk55MWxNVFV3TWpFNE5XVmlZamdpTENKcmFXUWlPaUowY21Ga1pWOXJaWGxmZGpJaUxDSnZiVzVsYldGdVlXZGxjbWxrSWpvNExDSndjbTlrZFdOMGN5STZleUprWlcxaGRDSTZleUp6ZEdGMGRYTWlPaUpoWTNScGRtVWlmU3dpYldZaU9uc2ljM1JoZEhWeklqb2lZV04wYVhabEluMTlMQ0pwYzNNaU9pSjBjbUZrWlY5c2IyZHBibDl6WlhKMmFXTmxJaXdpYzNWaUlqb2lVelUwT0RZME1qRTBJaXdpWlhod0lqb3hOelF4TXpJME9EQXhMQ0p1WW1ZaU9qRTNOREV5TXpneU1qRXNJbWxoZENJNk1UYzBNVEl6T0RJeU1Td2lhblJwSWpvaU1ERmhNamRsWmpBdFpqZGxaQzAwT0RCakxUazFaamt0WWpoaVpUQmlNVGt6TTJKaUlpd2lWRzlyWlc0aU9pSWlmUS53S3QtaktYVm5MMnlWeFdDSVFFYzhrTm12ZTYwbGRBU3ZCV3VuN3VQd2Y0M3Q2T2FjWmRyM0RvMDIybzVpNll2NlFXbTVZN2w0ZHh4TTVXd21GT0NyWFRBalVzY0JLUC1FOVVORWkxSS1tOUk1NU5leUgtLWtKSUJSbXpGVUZ5RG9vcEU2aHdFRHNwTk1JdUp0czJZeFFVc2plbnhTV19KNzdCbmw4UzVONTAiLCJBUEktS0VZIjoiOG1DZ2JqaUUiLCJpYXQiOjE3NDEyMzg0MDEsImV4cCI6MTc0MTMyNDgwMX0.F4eAeflNb6PzBnd02ktGJt_XVh_K6OpGlKjZgpHdgGX4lsahDfuAVJ8LZtfUnIPWmlGFVO_KBMBu8yBgY5aRYA'}, Request: {}, Response: HTTPSConnectionPool(host='apiconnect.angelone.in', port=443): Max retries exceeded with url: /rest/secure/angelbroking/portfolio/v1/getAllHolding?%7B%7D (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x112821f90>: Failed to resolve 'apiconnect.angelone.in' ([Errno 8] nodename nor servname provided, or not known)"))
[E 250306 11:10:24 smartConnect:234] Error occurred while making a GET request to https://apiconnect.angelone.in/rest/secure/angelbroking/portfolio/v1/getAllHolding. Headers: {'Content-type': 'application/json', 'X-ClientLocalIP': '127.0.0.1', 'X-ClientPublicIP': '106.193.147.98', 'X-MACAddress': '6a:e0:7a:cc:09:b3', 'Accept': 'application/json', 'X-PrivateKey': '8mCgbjiE', 'X-UserType': 'USER', 'X-SourceID': 'WEB', 'Authorization': 'Bearer eyJhbGciOiJIUzUxMiJ9.eyJ1c2VybmFtZSI6IlM1NDg2NDIxNCIsInJvbGVzIjowLCJ1c2VydHlwZSI6IlVTRVIiLCJ0b2tlbiI6ImV5SmhiR2NpT2lKU1V6STFOaUlzSW5SNWNDSTZJa3BYVkNKOS5leUoxYzJWeVgzUjVjR1VpT2lKamJHbGxiblFpTENKMGIydGxibDkwZVhCbElqb2lkSEpoWkdWZllXTmpaWE56WDNSdmEyVnVJaXdpWjIxZmFXUWlPamdzSW5OdmRYSmpaU0k2SWpNaUxDSmtaWFpwWTJWZmFXUWlPaUkxWkdVM1lqazROUzB3WVRrd0xUTmtNREl0T1RZMk55MWxNVFV3TWpFNE5XVmlZamdpTENKcmFXUWlPaUowY21Ga1pWOXJaWGxmZGpJaUxDSnZiVzVsYldGdVlXZGxjbWxrSWpvNExDSndjbTlrZFdOMGN5STZleUprWlcxaGRDSTZleUp6ZEdGMGRYTWlPaUpoWTNScGRtVWlmU3dpYldZaU9uc2ljM1JoZEhWeklqb2lZV04wYVhabEluMTlMQ0pwYzNNaU9pSjBjbUZrWlY5c2IyZHBibDl6WlhKMmFXTmxJaXdpYzNWaUlqb2lVelUwT0RZME1qRTBJaXdpWlhod0lqb3hOelF4TXpJME9EQXhMQ0p1WW1ZaU9qRTNOREV5TXpneU1qRXNJbWxoZENJNk1UYzBNVEl6T0RJeU1Td2lhblJwSWpvaU1ERmhNamRsWmpBdFpqZGxaQzAwT0RCakxUazFaamt0WWpoaVpUQmlNVGt6TTJKaUlpd2lWRzlyWlc0aU9pSWlmUS53S3QtaktYVm5MMnlWeFdDSVFFYzhrTm12ZTYwbGRBU3ZCV3VuN3VQd2Y0M3Q2T2FjWmRyM0RvMDIybzVpNll2NlFXbTVZN2w0ZHh4TTVXd21GT0NyWFRBalVzY0JLUC1FOVVORWkxSS1tOUk1NU5leUgtLWtKSUJSbXpGVUZ5RG9vcEU2aHdFRHNwTk1JdUp0czJZeFFVc2plbnhTV19KNzdCbmw4UzVONTAiLCJBUEktS0VZIjoiOG1DZ2JqaUUiLCJpYXQiOjE3NDEyMzg0MDEsImV4cCI6MTc0MTMyNDgwMX0.F4eAeflNb6PzBnd02ktGJt_XVh_K6OpGlKjZgpHdgGX4lsahDfuAVJ8LZtfUnIPWmlGFVO_KBMBu8yBgY5aRYA'}, Request: {}, Response: HTTPSConnectionPool(host='apiconnect.angelone.in', port=443): Max retries exceeded with url: /rest/secure/angelbroking/portfolio/v1/getAllHolding?%7B%7D (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x117924cd0>: Failed to resolve 'apiconnect.angelone.in' ([Errno 8] nodename nor servname provided, or not known)"))
[E 250306 11:10:35 smartConnect:234] Error occurred while making a GET request to https://apiconnect.angelone.in/rest/secure/angelbroking/portfolio/v1/getAllHolding. Headers: {'Content-type': 'application/json', 'X-ClientLocalIP': '127.0.0.1', 'X-ClientPublicIP': '106.193.147.98', 'X-MACAddress': '6a:e0:7a:cc:09:b3', 'Accept': 'application/json', 'X-PrivateKey': '8mCgbjiE', 'X-UserType': 'USER', 'X-SourceID': 'WEB', 'Authorization': 'Bearer eyJhbGciOiJIUzUxMiJ9.eyJ1c2VybmFtZSI6IlM1NDg2NDIxNCIsInJvbGVzIjowLCJ1c2VydHlwZSI6IlVTRVIiLCJ0b2tlbiI6ImV5SmhiR2NpT2lKU1V6STFOaUlzSW5SNWNDSTZJa3BYVkNKOS5leUoxYzJWeVgzUjVjR1VpT2lKamJHbGxiblFpTENKMGIydGxibDkwZVhCbElqb2lkSEpoWkdWZllXTmpaWE56WDNSdmEyVnVJaXdpWjIxZmFXUWlPamdzSW5OdmRYSmpaU0k2SWpNaUxDSmtaWFpwWTJWZmFXUWlPaUkxWkdVM1lqazROUzB3WVRrd0xUTmtNREl0T1RZMk55MWxNVFV3TWpFNE5XVmlZamdpTENKcmFXUWlPaUowY21Ga1pWOXJaWGxmZGpJaUxDSnZiVzVsYldGdVlXZGxjbWxrSWpvNExDSndjbTlrZFdOMGN5STZleUprWlcxaGRDSTZleUp6ZEdGMGRYTWlPaUpoWTNScGRtVWlmU3dpYldZaU9uc2ljM1JoZEhWeklqb2lZV04wYVhabEluMTlMQ0pwYzNNaU9pSjBjbUZrWlY5c2IyZHBibDl6WlhKMmFXTmxJaXdpYzNWaUlqb2lVelUwT0RZME1qRTBJaXdpWlhod0lqb3hOelF4TXpJME9EQXhMQ0p1WW1ZaU9qRTNOREV5TXpneU1qRXNJbWxoZENJNk1UYzBNVEl6T0RJeU1Td2lhblJwSWpvaU1ERmhNamRsWmpBdFpqZGxaQzAwT0RCakxUazFaamt0WWpoaVpUQmlNVGt6TTJKaUlpd2lWRzlyWlc0aU9pSWlmUS53S3QtaktYVm5MMnlWeFdDSVFFYzhrTm12ZTYwbGRBU3ZCV3VuN3VQd2Y0M3Q2T2FjWmRyM0RvMDIybzVpNll2NlFXbTVZN2w0ZHh4TTVXd21GT0NyWFRBalVzY0JLUC1FOVVORWkxSS1tOUk1NU5leUgtLWtKSUJSbXpGVUZ5RG9vcEU2aHdFRHNwTk1JdUp0czJZeFFVc2plbnhTV19KNzdCbmw4UzVONTAiLCJBUEktS0VZIjoiOG1DZ2JqaUUiLCJpYXQiOjE3NDEyMzg0MDEsImV4cCI6MTc0MTMyNDgwMX0.F4eAeflNb6PzBnd02ktGJt_XVh_K6OpGlKjZgpHdgGX4lsahDfuAVJ8LZtfUnIPWmlGFVO_KBMBu8yBgY5aRYA'}, Request: {}, Response: HTTPSConnectionPool(host='apiconnect.angelone.in', port=443): Max retries exceeded with url: /rest/secure/angelbroking/portfolio/v1/getAllHolding?%7B%7D (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x117925590>: Failed to resolve 'apiconnect.angelone.in' ([Errno 8] nodename nor servname provided, or not known)"))
[E 250306 11:10:35 smartConnect:234] Error occurred while making a GET request to https://apiconnect.angelone.in/rest/secure/angelbroking/portfolio/v1/getAllHolding. Headers: {'Content-type': 'application/json', 'X-ClientLocalIP': '127.0.0.1', 'X-ClientPublicIP': '106.193.147.98', 'X-MACAddress': '6a:e0:7a:cc:09:b3', 'Accept': 'application/json', 'X-PrivateKey': '8mCgbjiE', 'X-UserType': 'USER', 'X-SourceID': 'WEB', 'Authorization': 'Bearer eyJhbGciOiJIUzUxMiJ9.eyJ1c2VybmFtZSI6IlM1NDg2NDIxNCIsInJvbGVzIjowLCJ1c2VydHlwZSI6IlVTRVIiLCJ0b2tlbiI6ImV5SmhiR2NpT2lKU1V6STFOaUlzSW5SNWNDSTZJa3BYVkNKOS5leUoxYzJWeVgzUjVjR1VpT2lKamJHbGxiblFpTENKMGIydGxibDkwZVhCbElqb2lkSEpoWkdWZllXTmpaWE56WDNSdmEyVnVJaXdpWjIxZmFXUWlPamdzSW5OdmRYSmpaU0k2SWpNaUxDSmtaWFpwWTJWZmFXUWlPaUkxWkdVM1lqazROUzB3WVRrd0xUTmtNREl0T1RZMk55MWxNVFV3TWpFNE5XVmlZamdpTENKcmFXUWlPaUowY21Ga1pWOXJaWGxmZGpJaUxDSnZiVzVsYldGdVlXZGxjbWxrSWpvNExDSndjbTlrZFdOMGN5STZleUprWlcxaGRDSTZleUp6ZEdGMGRYTWlPaUpoWTNScGRtVWlmU3dpYldZaU9uc2ljM1JoZEhWeklqb2lZV04wYVhabEluMTlMQ0pwYzNNaU9pSjBjbUZrWlY5c2IyZHBibDl6WlhKMmFXTmxJaXdpYzNWaUlqb2lVelUwT0RZME1qRTBJaXdpWlhod0lqb3hOelF4TXpJME9EQXhMQ0p1WW1ZaU9qRTNOREV5TXpneU1qRXNJbWxoZENJNk1UYzBNVEl6T0RJeU1Td2lhblJwSWpvaU1ERmhNamRsWmpBdFpqZGxaQzAwT0RCakxUazFaamt0WWpoaVpUQmlNVGt6TTJKaUlpd2lWRzlyWlc0aU9pSWlmUS53S3QtaktYVm5MMnlWeFdDSVFFYzhrTm12ZTYwbGRBU3ZCV3VuN3VQd2Y0M3Q2T2FjWmRyM0RvMDIybzVpNll2NlFXbTVZN2w0ZHh4TTVXd21GT0NyWFRBalVzY0JLUC1FOVVORWkxSS1tOUk1NU5leUgtLWtKSUJSbXpGVUZ5RG9vcEU2aHdFRHNwTk1JdUp0czJZeFFVc2plbnhTV19KNzdCbmw4UzVONTAiLCJBUEktS0VZIjoiOG1DZ2JqaUUiLCJpYXQiOjE3NDEyMzg0MDEsImV4cCI6MTc0MTMyNDgwMX0.F4eAeflNb6PzBnd02ktGJt_XVh_K6OpGlKjZgpHdgGX4lsahDfuAVJ8LZtfUnIPWmlGFVO_KBMBu8yBgY5aRYA'}, Request: {}, Response: HTTPSConnectionPool(host='apiconnect.angelone.in', port=443): Max retries exceeded with url: /rest/secure/angelbroking/portfolio/v1/getAllHolding?%7B%7D (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1179251d0>: Failed to resolve 'apiconnect.angelone.in' ([Errno 8] nodename nor servname provided, or not known)"))
[E 250306 11:10:35 smartConnect:234] Error occurred while making a GET request to https://apiconnect.angelone.in/rest/secure/angelbroking/portfolio/v1/getAllHolding. Headers: {'Content-type': 'application/json', 'X-ClientLocalIP': '127.0.0.1', 'X-ClientPublicIP': '106.193.147.98', 'X-MACAddress': '6a:e0:7a:cc:09:b3', 'Accept': 'application/json', 'X-PrivateKey': '8mCgbjiE', 'X-UserType': 'USER', 'X-SourceID': 'WEB', 'Authorization': 'Bearer eyJhbGciOiJIUzUxMiJ9.eyJ1c2VybmFtZSI6IlM1NDg2NDIxNCIsInJvbGVzIjowLCJ1c2VydHlwZSI6IlVTRVIiLCJ0b2tlbiI6ImV5SmhiR2NpT2lKU1V6STFOaUlzSW5SNWNDSTZJa3BYVkNKOS5leUoxYzJWeVgzUjVjR1VpT2lKamJHbGxiblFpTENKMGIydGxibDkwZVhCbElqb2lkSEpoWkdWZllXTmpaWE56WDNSdmEyVnVJaXdpWjIxZmFXUWlPamdzSW5OdmRYSmpaU0k2SWpNaUxDSmtaWFpwWTJWZmFXUWlPaUkxWkdVM1lqazROUzB3WVRrd0xUTmtNREl0T1RZMk55MWxNVFV3TWpFNE5XVmlZamdpTENKcmFXUWlPaUowY21Ga1pWOXJaWGxmZGpJaUxDSnZiVzVsYldGdVlXZGxjbWxrSWpvNExDSndjbTlrZFdOMGN5STZleUprWlcxaGRDSTZleUp6ZEdGMGRYTWlPaUpoWTNScGRtVWlmU3dpYldZaU9uc2ljM1JoZEhWeklqb2lZV04wYVhabEluMTlMQ0pwYzNNaU9pSjBjbUZrWlY5c2IyZHBibDl6WlhKMmFXTmxJaXdpYzNWaUlqb2lVelUwT0RZME1qRTBJaXdpWlhod0lqb3hOelF4TXpJME9EQXhMQ0p1WW1ZaU9qRTNOREV5TXpneU1qRXNJbWxoZENJNk1UYzBNVEl6T0RJeU1Td2lhblJwSWpvaU1ERmhNamRsWmpBdFpqZGxaQzAwT0RCakxUazFaamt0WWpoaVpUQmlNVGt6TTJKaUlpd2lWRzlyWlc0aU9pSWlmUS53S3QtaktYVm5MMnlWeFdDSVFFYzhrTm12ZTYwbGRBU3ZCV3VuN3VQd2Y0M3Q2T2FjWmRyM0RvMDIybzVpNll2NlFXbTVZN2w0ZHh4TTVXd21GT0NyWFRBalVzY0JLUC1FOVVORWkxSS1tOUk1NU5leUgtLWtKSUJSbXpGVUZ5RG9vcEU2aHdFRHNwTk1JdUp0czJZeFFVc2plbnhTV19KNzdCbmw4UzVONTAiLCJBUEktS0VZIjoiOG1DZ2JqaUUiLCJpYXQiOjE3NDEyMzg0MDEsImV4cCI6MTc0MTMyNDgwMX0.F4eAeflNb6PzBnd02ktGJt_XVh_K6OpGlKjZgpHdgGX4lsahDfuAVJ8LZtfUnIPWmlGFVO_KBMBu8yBgY5aRYA'}, Request: {}, Response: HTTPSConnectionPool(host='apiconnect.angelone.in', port=443): Max retries exceeded with url: /rest/secure/angelbroking/portfolio/v1/getAllHolding?%7B%7D (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x117925450>: Failed to resolve 'apiconnect.angelone.in' ([Errno 8] nodename nor servname provided, or not known)"))
[E 250306 11:10:45 smartConnect:234] Error occurred while making a GET request to https://apiconnect.angelone.in/rest/secure/angelbroking/portfolio/v1/getAllHolding. Headers: {'Content-type': 'application/json', 'X-ClientLocalIP': '127.0.0.1', 'X-ClientPublicIP': '106.193.147.98', 'X-MACAddress': '6a:e0:7a:cc:09:b3', 'Accept': 'application/json', 'X-PrivateKey': '8mCgbjiE', 'X-UserType': 'USER', 'X-SourceID': 'WEB', 'Authorization': 'Bearer eyJhbGciOiJIUzUxMiJ9.eyJ1c2VybmFtZSI6IlM1NDg2NDIxNCIsInJvbGVzIjowLCJ1c2VydHlwZSI6IlVTRVIiLCJ0b2tlbiI6ImV5SmhiR2NpT2lKU1V6STFOaUlzSW5SNWNDSTZJa3BYVkNKOS5leUoxYzJWeVgzUjVjR1VpT2lKamJHbGxiblFpTENKMGIydGxibDkwZVhCbElqb2lkSEpoWkdWZllXTmpaWE56WDNSdmEyVnVJaXdpWjIxZmFXUWlPamdzSW5OdmRYSmpaU0k2SWpNaUxDSmtaWFpwWTJWZmFXUWlPaUkxWkdVM1lqazROUzB3WVRrd0xUTmtNREl0T1RZMk55MWxNVFV3TWpFNE5XVmlZamdpTENKcmFXUWlPaUowY21Ga1pWOXJaWGxmZGpJaUxDSnZiVzVsYldGdVlXZGxjbWxrSWpvNExDSndjbTlrZFdOMGN5STZleUprWlcxaGRDSTZleUp6ZEdGMGRYTWlPaUpoWTNScGRtVWlmU3dpYldZaU9uc2ljM1JoZEhWeklqb2lZV04wYVhabEluMTlMQ0pwYzNNaU9pSjBjbUZrWlY5c2IyZHBibDl6WlhKMmFXTmxJaXdpYzNWaUlqb2lVelUwT0RZME1qRTBJaXdpWlhod0lqb3hOelF4TXpJME9EQXhMQ0p1WW1ZaU9qRTNOREV5TXpneU1qRXNJbWxoZENJNk1UYzBNVEl6T0RJeU1Td2lhblJwSWpvaU1ERmhNamRsWmpBdFpqZGxaQzAwT0RCakxUazFaamt0WWpoaVpUQmlNVGt6TTJKaUlpd2lWRzlyWlc0aU9pSWlmUS53S3QtaktYVm5MMnlWeFdDSVFFYzhrTm12ZTYwbGRBU3ZCV3VuN3VQd2Y0M3Q2T2FjWmRyM0RvMDIybzVpNll2NlFXbTVZN2w0ZHh4TTVXd21GT0NyWFRBalVzY0JLUC1FOVVORWkxSS1tOUk1NU5leUgtLWtKSUJSbXpGVUZ5RG9vcEU2aHdFRHNwTk1JdUp0czJZeFFVc2plbnhTV19KNzdCbmw4UzVONTAiLCJBUEktS0VZIjoiOG1DZ2JqaUUiLCJpYXQiOjE3NDEyMzg0MDEsImV4cCI6MTc0MTMyNDgwMX0.F4eAeflNb6PzBnd02ktGJt_XVh_K6OpGlKjZgpHdgGX4lsahDfuAVJ8LZtfUnIPWmlGFVO_KBMBu8yBgY5aRYA'}, Request: {}, Response: HTTPSConnectionPool(host='apiconnect.angelone.in', port=443): Max retries exceeded with url: /rest/secure/angelbroking/portfolio/v1/getAllHolding?%7B%7D (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1128211d0>: Failed to resolve 'apiconnect.angelone.in' ([Errno 8] nodename nor servname provided, or not known)"))
[E 250306 11:10:45 smartConnect:234] Error occurred while making a GET request to https://apiconnect.angelone.in/rest/secure/angelbroking/portfolio/v1/getAllHolding. Headers: {'Content-type': 'application/json', 'X-ClientLocalIP': '127.0.0.1', 'X-ClientPublicIP': '106.193.147.98', 'X-MACAddress': '6a:e0:7a:cc:09:b3', 'Accept': 'application/json', 'X-PrivateKey': '8mCgbjiE', 'X-UserType': 'USER', 'X-SourceID': 'WEB', 'Authorization': 'Bearer eyJhbGciOiJIUzUxMiJ9.eyJ1c2VybmFtZSI6IlM1NDg2NDIxNCIsInJvbGVzIjowLCJ1c2VydHlwZSI6IlVTRVIiLCJ0b2tlbiI6ImV5SmhiR2NpT2lKU1V6STFOaUlzSW5SNWNDSTZJa3BYVkNKOS5leUoxYzJWeVgzUjVjR1VpT2lKamJHbGxiblFpTENKMGIydGxibDkwZVhCbElqb2lkSEpoWkdWZllXTmpaWE56WDNSdmEyVnVJaXdpWjIxZmFXUWlPamdzSW5OdmRYSmpaU0k2SWpNaUxDSmtaWFpwWTJWZmFXUWlPaUkxWkdVM1lqazROUzB3WVRrd0xUTmtNREl0T1RZMk55MWxNVFV3TWpFNE5XVmlZamdpTENKcmFXUWlPaUowY21Ga1pWOXJaWGxmZGpJaUxDSnZiVzVsYldGdVlXZGxjbWxrSWpvNExDSndjbTlrZFdOMGN5STZleUprWlcxaGRDSTZleUp6ZEdGMGRYTWlPaUpoWTNScGRtVWlmU3dpYldZaU9uc2ljM1JoZEhWeklqb2lZV04wYVhabEluMTlMQ0pwYzNNaU9pSjBjbUZrWlY5c2IyZHBibDl6WlhKMmFXTmxJaXdpYzNWaUlqb2lVelUwT0RZME1qRTBJaXdpWlhod0lqb3hOelF4TXpJME9EQXhMQ0p1WW1ZaU9qRTNOREV5TXpneU1qRXNJbWxoZENJNk1UYzBNVEl6T0RJeU1Td2lhblJwSWpvaU1ERmhNamRsWmpBdFpqZGxaQzAwT0RCakxUazFaamt0WWpoaVpUQmlNVGt6TTJKaUlpd2lWRzlyWlc0aU9pSWlmUS53S3QtaktYVm5MMnlWeFdDSVFFYzhrTm12ZTYwbGRBU3ZCV3VuN3VQd2Y0M3Q2T2FjWmRyM0RvMDIybzVpNll2NlFXbTVZN2w0ZHh4TTVXd21GT0NyWFRBalVzY0JLUC1FOVVORWkxSS1tOUk1NU5leUgtLWtKSUJSbXpGVUZ5RG9vcEU2aHdFRHNwTk1JdUp0czJZeFFVc2plbnhTV19KNzdCbmw4UzVONTAiLCJBUEktS0VZIjoiOG1DZ2JqaUUiLCJpYXQiOjE3NDEyMzg0MDEsImV4cCI6MTc0MTMyNDgwMX0.F4eAeflNb6PzBnd02ktGJt_XVh_K6OpGlKjZgpHdgGX4lsahDfuAVJ8LZtfUnIPWmlGFVO_KBMBu8yBgY5aRYA'}, Request: {}, Response: HTTPSConnectionPool(host='apiconnect.angelone.in', port=443): Max retries exceeded with url: /rest/secure/angelbroking/portfolio/v1/getAllHolding?%7B%7D (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x112821f90>: Failed to resolve 'apiconnect.angelone.in' ([Errno 8] nodename nor servname provided, or not known)"))
[E 250306 11:10:56 smartConnect:234] Error occurred while making a GET request to https://apiconnect.angelone.in/rest/secure/angelbroking/portfolio/v1/getAllHolding. Headers: {'Content-type': 'application/json', 'X-ClientLocalIP': '127.0.0.1', 'X-ClientPublicIP': '106.193.147.98', 'X-MACAddress': '6a:e0:7a:cc:09:b3', 'Accept': 'application/json', 'X-PrivateKey': '8mCgbjiE', 'X-UserType': 'USER', 'X-SourceID': 'WEB', 'Authorization': 'Bearer eyJhbGciOiJIUzUxMiJ9.eyJ1c2VybmFtZSI6IlM1NDg2NDIxNCIsInJvbGVzIjowLCJ1c2VydHlwZSI6IlVTRVIiLCJ0b2tlbiI6ImV5SmhiR2NpT2lKU1V6STFOaUlzSW5SNWNDSTZJa3BYVkNKOS5leUoxYzJWeVgzUjVjR1VpT2lKamJHbGxiblFpTENKMGIydGxibDkwZVhCbElqb2lkSEpoWkdWZllXTmpaWE56WDNSdmEyVnVJaXdpWjIxZmFXUWlPamdzSW5OdmRYSmpaU0k2SWpNaUxDSmtaWFpwWTJWZmFXUWlPaUkxWkdVM1lqazROUzB3WVRrd0xUTmtNREl0T1RZMk55MWxNVFV3TWpFNE5XVmlZamdpTENKcmFXUWlPaUowY21Ga1pWOXJaWGxmZGpJaUxDSnZiVzVsYldGdVlXZGxjbWxrSWpvNExDSndjbTlrZFdOMGN5STZleUprWlcxaGRDSTZleUp6ZEdGMGRYTWlPaUpoWTNScGRtVWlmU3dpYldZaU9uc2ljM1JoZEhWeklqb2lZV04wYVhabEluMTlMQ0pwYzNNaU9pSjBjbUZrWlY5c2IyZHBibDl6WlhKMmFXTmxJaXdpYzNWaUlqb2lVelUwT0RZME1qRTBJaXdpWlhod0lqb3hOelF4TXpJME9EQXhMQ0p1WW1ZaU9qRTNOREV5TXpneU1qRXNJbWxoZENJNk1UYzBNVEl6T0RJeU1Td2lhblJwSWpvaU1ERmhNamRsWmpBdFpqZGxaQzAwT0RCakxUazFaamt0WWpoaVpUQmlNVGt6TTJKaUlpd2lWRzlyWlc0aU9pSWlmUS53S3QtaktYVm5MMnlWeFdDSVFFYzhrTm12ZTYwbGRBU3ZCV3VuN3VQd2Y0M3Q2T2FjWmRyM0RvMDIybzVpNll2NlFXbTVZN2w0ZHh4TTVXd21GT0NyWFRBalVzY0JLUC1FOVVORWkxSS1tOUk1NU5leUgtLWtKSUJSbXpGVUZ5RG9vcEU2aHdFRHNwTk1JdUp0czJZeFFVc2plbnhTV19KNzdCbmw4UzVONTAiLCJBUEktS0VZIjoiOG1DZ2JqaUUiLCJpYXQiOjE3NDEyMzg0MDEsImV4cCI6MTc0MTMyNDgwMX0.F4eAeflNb6PzBnd02ktGJt_XVh_K6OpGlKjZgpHdgGX4lsahDfuAVJ8LZtfUnIPWmlGFVO_KBMBu8yBgY5aRYA'}, Request: {}, Response: HTTPSConnectionPool(host='apiconnect.angelone.in', port=443): Max retries exceeded with url: /rest/secure/angelbroking/portfolio/v1/getAllHolding?%7B%7D (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x117925e50>: Failed to resolve 'apiconnect.angelone.in' ([Errno 8] nodename nor servname provided, or not known)"))
[E 250306 11:11:06 smartConnect:234] Error occurred while making a GET request to https://apiconnect.angelone.in/rest/secure/angelbroking/portfolio/v1/getAllHolding. Headers: {'Content-type': 'application/json', 'X-ClientLocalIP': '127.0.0.1', 'X-ClientPublicIP': '106.193.147.98', 'X-MACAddress': '6a:e0:7a:cc:09:b3', 'Accept': 'application/json', 'X-PrivateKey': '8mCgbjiE', 'X-UserType': 'USER', 'X-SourceID': 'WEB', 'Authorization': 'Bearer eyJhbGciOiJIUzUxMiJ9.eyJ1c2VybmFtZSI6IlM1NDg2NDIxNCIsInJvbGVzIjowLCJ1c2VydHlwZSI6IlVTRVIiLCJ0b2tlbiI6ImV5SmhiR2NpT2lKU1V6STFOaUlzSW5SNWNDSTZJa3BYVkNKOS5leUoxYzJWeVgzUjVjR1VpT2lKamJHbGxiblFpTENKMGIydGxibDkwZVhCbElqb2lkSEpoWkdWZllXTmpaWE56WDNSdmEyVnVJaXdpWjIxZmFXUWlPamdzSW5OdmRYSmpaU0k2SWpNaUxDSmtaWFpwWTJWZmFXUWlPaUkxWkdVM1lqazROUzB3WVRrd0xUTmtNREl0T1RZMk55MWxNVFV3TWpFNE5XVmlZamdpTENKcmFXUWlPaUowY21Ga1pWOXJaWGxmZGpJaUxDSnZiVzVsYldGdVlXZGxjbWxrSWpvNExDSndjbTlrZFdOMGN5STZleUprWlcxaGRDSTZleUp6ZEdGMGRYTWlPaUpoWTNScGRtVWlmU3dpYldZaU9uc2ljM1JoZEhWeklqb2lZV04wYVhabEluMTlMQ0pwYzNNaU9pSjBjbUZrWlY5c2IyZHBibDl6WlhKMmFXTmxJaXdpYzNWaUlqb2lVelUwT0RZME1qRTBJaXdpWlhod0lqb3hOelF4TXpJME9EQXhMQ0p1WW1ZaU9qRTNOREV5TXpneU1qRXNJbWxoZENJNk1UYzBNVEl6T0RJeU1Td2lhblJwSWpvaU1ERmhNamRsWmpBdFpqZGxaQzAwT0RCakxUazFaamt0WWpoaVpUQmlNVGt6TTJKaUlpd2lWRzlyWlc0aU9pSWlmUS53S3QtaktYVm5MMnlWeFdDSVFFYzhrTm12ZTYwbGRBU3ZCV3VuN3VQd2Y0M3Q2T2FjWmRyM0RvMDIybzVpNll2NlFXbTVZN2w0ZHh4TTVXd21GT0NyWFRBalVzY0JLUC1FOVVORWkxSS1tOUk1NU5leUgtLWtKSUJSbXpGVUZ5RG9vcEU2aHdFRHNwTk1JdUp0czJZeFFVc2plbnhTV19KNzdCbmw4UzVONTAiLCJBUEktS0VZIjoiOG1DZ2JqaUUiLCJpYXQiOjE3NDEyMzg0MDEsImV4cCI6MTc0MTMyNDgwMX0.F4eAeflNb6PzBnd02ktGJt_XVh_K6OpGlKjZgpHdgGX4lsahDfuAVJ8LZtfUnIPWmlGFVO_KBMBu8yBgY5aRYA'}, Request: {}, Response: HTTPSConnectionPool(host='apiconnect.angelone.in', port=443): Max retries exceeded with url: /rest/secure/angelbroking/portfolio/v1/getAllHolding?%7B%7D (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x112823250>: Failed to resolve 'apiconnect.angelone.in' ([Errno 8] nodename nor servname provided, or not known)"))
[E 250306 11:11:06 smartConnect:234] Error occurred while making a GET request to https://apiconnect.angelone.in/rest/secure/angelbroking/portfolio/v1/getAllHolding. Headers: {'Content-type': 'application/json', 'X-ClientLocalIP': '127.0.0.1', 'X-ClientPublicIP': '106.193.147.98', 'X-MACAddress': '6a:e0:7a:cc:09:b3', 'Accept': 'application/json', 'X-PrivateKey': '8mCgbjiE', 'X-UserType': 'USER', 'X-SourceID': 'WEB', 'Authorization': 'Bearer eyJhbGciOiJIUzUxMiJ9.eyJ1c2VybmFtZSI6IlM1NDg2NDIxNCIsInJvbGVzIjowLCJ1c2VydHlwZSI6IlVTRVIiLCJ0b2tlbiI6ImV5SmhiR2NpT2lKU1V6STFOaUlzSW5SNWNDSTZJa3BYVkNKOS5leUoxYzJWeVgzUjVjR1VpT2lKamJHbGxiblFpTENKMGIydGxibDkwZVhCbElqb2lkSEpoWkdWZllXTmpaWE56WDNSdmEyVnVJaXdpWjIxZmFXUWlPamdzSW5OdmRYSmpaU0k2SWpNaUxDSmtaWFpwWTJWZmFXUWlPaUkxWkdVM1lqazROUzB3WVRrd0xUTmtNREl0T1RZMk55MWxNVFV3TWpFNE5XVmlZamdpTENKcmFXUWlPaUowY21Ga1pWOXJaWGxmZGpJaUxDSnZiVzVsYldGdVlXZGxjbWxrSWpvNExDSndjbTlrZFdOMGN5STZleUprWlcxaGRDSTZleUp6ZEdGMGRYTWlPaUpoWTNScGRtVWlmU3dpYldZaU9uc2ljM1JoZEhWeklqb2lZV04wYVhabEluMTlMQ0pwYzNNaU9pSjBjbUZrWlY5c2IyZHBibDl6WlhKMmFXTmxJaXdpYzNWaUlqb2lVelUwT0RZME1qRTBJaXdpWlhod0lqb3hOelF4TXpJME9EQXhMQ0p1WW1ZaU9qRTNOREV5TXpneU1qRXNJbWxoZENJNk1UYzBNVEl6T0RJeU1Td2lhblJwSWpvaU1ERmhNamRsWmpBdFpqZGxaQzAwT0RCakxUazFaamt0WWpoaVpUQmlNVGt6TTJKaUlpd2lWRzlyWlc0aU9pSWlmUS53S3QtaktYVm5MMnlWeFdDSVFFYzhrTm12ZTYwbGRBU3ZCV3VuN3VQd2Y0M3Q2T2FjWmRyM0RvMDIybzVpNll2NlFXbTVZN2w0ZHh4TTVXd21GT0NyWFRBalVzY0JLUC1FOVVORWkxSS1tOUk1NU5leUgtLWtKSUJSbXpGVUZ5RG9vcEU2aHdFRHNwTk1JdUp0czJZeFFVc2plbnhTV19KNzdCbmw4UzVONTAiLCJBUEktS0VZIjoiOG1DZ2JqaUUiLCJpYXQiOjE3NDEyMzg0MDEsImV4cCI6MTc0MTMyNDgwMX0.F4eAeflNb6PzBnd02ktGJt_XVh_K6OpGlKjZgpHdgGX4lsahDfuAVJ8LZtfUnIPWmlGFVO_KBMBu8yBgY5aRYA'}, Request: {}, Response: HTTPSConnectionPool(host='apiconnect.angelone.in', port=443): Max retries exceeded with url: /rest/secure/angelbroking/portfolio/v1/getAllHolding?%7B%7D (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1062fb750>: Failed to resolve 'apiconnect.angelone.in' ([Errno 8] nodename nor servname provided, or not known)"))
[E 250306 11:11:06 smartConnect:234] Error occurred while making a GET request to https://apiconnect.angelone.in/rest/secure/angelbroking/portfolio/v1/getAllHolding. Headers: {'Content-type': 'application/json', 'X-ClientLocalIP': '127.0.0.1', 'X-ClientPublicIP': '106.193.147.98', 'X-MACAddress': '6a:e0:7a:cc:09:b3', 'Accept': 'application/json', 'X-PrivateKey': '8mCgbjiE', 'X-UserType': 'USER', 'X-SourceID': 'WEB', 'Authorization': 'Bearer eyJhbGciOiJIUzUxMiJ9.eyJ1c2VybmFtZSI6IlM1NDg2NDIxNCIsInJvbGVzIjowLCJ1c2VydHlwZSI6IlVTRVIiLCJ0b2tlbiI6ImV5SmhiR2NpT2lKU1V6STFOaUlzSW5SNWNDSTZJa3BYVkNKOS5leUoxYzJWeVgzUjVjR1VpT2lKamJHbGxiblFpTENKMGIydGxibDkwZVhCbElqb2lkSEpoWkdWZllXTmpaWE56WDNSdmEyVnVJaXdpWjIxZmFXUWlPamdzSW5OdmRYSmpaU0k2SWpNaUxDSmtaWFpwWTJWZmFXUWlPaUkxWkdVM1lqazROUzB3WVRrd0xUTmtNREl0T1RZMk55MWxNVFV3TWpFNE5XVmlZamdpTENKcmFXUWlPaUowY21Ga1pWOXJaWGxmZGpJaUxDSnZiVzVsYldGdVlXZGxjbWxrSWpvNExDSndjbTlrZFdOMGN5STZleUprWlcxaGRDSTZleUp6ZEdGMGRYTWlPaUpoWTNScGRtVWlmU3dpYldZaU9uc2ljM1JoZEhWeklqb2lZV04wYVhabEluMTlMQ0pwYzNNaU9pSjBjbUZrWlY5c2IyZHBibDl6WlhKMmFXTmxJaXdpYzNWaUlqb2lVelUwT0RZME1qRTBJaXdpWlhod0lqb3hOelF4TXpJME9EQXhMQ0p1WW1ZaU9qRTNOREV5TXpneU1qRXNJbWxoZENJNk1UYzBNVEl6T0RJeU1Td2lhblJwSWpvaU1ERmhNamRsWmpBdFpqZGxaQzAwT0RCakxUazFaamt0WWpoaVpUQmlNVGt6TTJKaUlpd2lWRzlyWlc0aU9pSWlmUS53S3QtaktYVm5MMnlWeFdDSVFFYzhrTm12ZTYwbGRBU3ZCV3VuN3VQd2Y0M3Q2T2FjWmRyM0RvMDIybzVpNll2NlFXbTVZN2w0ZHh4TTVXd21GT0NyWFRBalVzY0JLUC1FOVVORWkxSS1tOUk1NU5leUgtLWtKSUJSbXpGVUZ5RG9vcEU2aHdFRHNwTk1JdUp0czJZeFFVc2plbnhTV19KNzdCbmw4UzVONTAiLCJBUEktS0VZIjoiOG1DZ2JqaUUiLCJpYXQiOjE3NDEyMzg0MDEsImV4cCI6MTc0MTMyNDgwMX0.F4eAeflNb6PzBnd02ktGJt_XVh_K6OpGlKjZgpHdgGX4lsahDfuAVJ8LZtfUnIPWmlGFVO_KBMBu8yBgY5aRYA'}, Request: {}, Response: HTTPSConnectionPool(host='apiconnect.angelone.in', port=443): Max retries exceeded with url: /rest/secure/angelbroking/portfolio/v1/getAllHolding?%7B%7D (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x112822e90>: Failed to resolve 'apiconnect.angelone.in' ([Errno 8] nodename nor servname provided, or not known)"))
[E 250306 11:11:06 smartConnect:234] Error occurred while making a GET request to https://apiconnect.angelone.in/rest/secure/angelbroking/portfolio/v1/getAllHolding. Headers: {'Content-type': 'application/json', 'X-ClientLocalIP': '127.0.0.1', 'X-ClientPublicIP': '106.193.147.98', 'X-MACAddress': '6a:e0:7a:cc:09:b3', 'Accept': 'application/json', 'X-PrivateKey': '8mCgbjiE', 'X-UserType': 'USER', 'X-SourceID': 'WEB', 'Authorization': 'Bearer eyJhbGciOiJIUzUxMiJ9.eyJ1c2VybmFtZSI6IlM1NDg2NDIxNCIsInJvbGVzIjowLCJ1c2VydHlwZSI6IlVTRVIiLCJ0b2tlbiI6ImV5SmhiR2NpT2lKU1V6STFOaUlzSW5SNWNDSTZJa3BYVkNKOS5leUoxYzJWeVgzUjVjR1VpT2lKamJHbGxiblFpTENKMGIydGxibDkwZVhCbElqb2lkSEpoWkdWZllXTmpaWE56WDNSdmEyVnVJaXdpWjIxZmFXUWlPamdzSW5OdmRYSmpaU0k2SWpNaUxDSmtaWFpwWTJWZmFXUWlPaUkxWkdVM1lqazROUzB3WVRrd0xUTmtNREl0T1RZMk55MWxNVFV3TWpFNE5XVmlZamdpTENKcmFXUWlPaUowY21Ga1pWOXJaWGxmZGpJaUxDSnZiVzVsYldGdVlXZGxjbWxrSWpvNExDSndjbTlrZFdOMGN5STZleUprWlcxaGRDSTZleUp6ZEdGMGRYTWlPaUpoWTNScGRtVWlmU3dpYldZaU9uc2ljM1JoZEhWeklqb2lZV04wYVhabEluMTlMQ0pwYzNNaU9pSjBjbUZrWlY5c2IyZHBibDl6WlhKMmFXTmxJaXdpYzNWaUlqb2lVelUwT0RZME1qRTBJaXdpWlhod0lqb3hOelF4TXpJME9EQXhMQ0p1WW1ZaU9qRTNOREV5TXpneU1qRXNJbWxoZENJNk1UYzBNVEl6T0RJeU1Td2lhblJwSWpvaU1ERmhNamRsWmpBdFpqZGxaQzAwT0RCakxUazFaamt0WWpoaVpUQmlNVGt6TTJKaUlpd2lWRzlyWlc0aU9pSWlmUS53S3QtaktYVm5MMnlWeFdDSVFFYzhrTm12ZTYwbGRBU3ZCV3VuN3VQd2Y0M3Q2T2FjWmRyM0RvMDIybzVpNll2NlFXbTVZN2w0ZHh4TTVXd21GT0NyWFRBalVzY0JLUC1FOVVORWkxSS1tOUk1NU5leUgtLWtKSUJSbXpGVUZ5RG9vcEU2aHdFRHNwTk1JdUp0czJZeFFVc2plbnhTV19KNzdCbmw4UzVONTAiLCJBUEktS0VZIjoiOG1DZ2JqaUUiLCJpYXQiOjE3NDEyMzg0MDEsImV4cCI6MTc0MTMyNDgwMX0.F4eAeflNb6PzBnd02ktGJt_XVh_K6OpGlKjZgpHdgGX4lsahDfuAVJ8LZtfUnIPWmlGFVO_KBMBu8yBgY5aRYA'}, Request: {}, Response: HTTPSConnectionPool(host='apiconnect.angelone.in', port=443): Max retries exceeded with url: /rest/secure/angelbroking/portfolio/v1/getAllHolding?%7B%7D (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1179260d0>: Failed to resolve 'apiconnect.angelone.in' ([Errno 8] nodename nor servname provided, or not known)"))
[E 250306 11:11:06 smartConnect:234] Error occurred while making a GET request to https://apiconnect.angelone.in/rest/secure/angelbroking/portfolio/v1/getAllHolding. Headers: {'Content-type': 'application/json', 'X-ClientLocalIP': '127.0.0.1', 'X-ClientPublicIP': '106.193.147.98', 'X-MACAddress': '6a:e0:7a:cc:09:b3', 'Accept': 'application/json', 'X-PrivateKey': '8mCgbjiE', 'X-UserType': 'USER', 'X-SourceID': 'WEB', 'Authorization': 'Bearer eyJhbGciOiJIUzUxMiJ9.eyJ1c2VybmFtZSI6IlM1NDg2NDIxNCIsInJvbGVzIjowLCJ1c2VydHlwZSI6IlVTRVIiLCJ0b2tlbiI6ImV5SmhiR2NpT2lKU1V6STFOaUlzSW5SNWNDSTZJa3BYVkNKOS5leUoxYzJWeVgzUjVjR1VpT2lKamJHbGxiblFpTENKMGIydGxibDkwZVhCbElqb2lkSEpoWkdWZllXTmpaWE56WDNSdmEyVnVJaXdpWjIxZmFXUWlPamdzSW5OdmRYSmpaU0k2SWpNaUxDSmtaWFpwWTJWZmFXUWlPaUkxWkdVM1lqazROUzB3WVRrd0xUTmtNREl0T1RZMk55MWxNVFV3TWpFNE5XVmlZamdpTENKcmFXUWlPaUowY21Ga1pWOXJaWGxmZGpJaUxDSnZiVzVsYldGdVlXZGxjbWxrSWpvNExDSndjbTlrZFdOMGN5STZleUprWlcxaGRDSTZleUp6ZEdGMGRYTWlPaUpoWTNScGRtVWlmU3dpYldZaU9uc2ljM1JoZEhWeklqb2lZV04wYVhabEluMTlMQ0pwYzNNaU9pSjBjbUZrWlY5c2IyZHBibDl6WlhKMmFXTmxJaXdpYzNWaUlqb2lVelUwT0RZME1qRTBJaXdpWlhod0lqb3hOelF4TXpJME9EQXhMQ0p1WW1ZaU9qRTNOREV5TXpneU1qRXNJbWxoZENJNk1UYzBNVEl6T0RJeU1Td2lhblJwSWpvaU1ERmhNamRsWmpBdFpqZGxaQzAwT0RCakxUazFaamt0WWpoaVpUQmlNVGt6TTJKaUlpd2lWRzlyWlc0aU9pSWlmUS53S3QtaktYVm5MMnlWeFdDSVFFYzhrTm12ZTYwbGRBU3ZCV3VuN3VQd2Y0M3Q2T2FjWmRyM0RvMDIybzVpNll2NlFXbTVZN2w0ZHh4TTVXd21GT0NyWFRBalVzY0JLUC1FOVVORWkxSS1tOUk1NU5leUgtLWtKSUJSbXpGVUZ5RG9vcEU2aHdFRHNwTk1JdUp0czJZeFFVc2plbnhTV19KNzdCbmw4UzVONTAiLCJBUEktS0VZIjoiOG1DZ2JqaUUiLCJpYXQiOjE3NDEyMzg0MDEsImV4cCI6MTc0MTMyNDgwMX0.F4eAeflNb6PzBnd02ktGJt_XVh_K6OpGlKjZgpHdgGX4lsahDfuAVJ8LZtfUnIPWmlGFVO_KBMBu8yBgY5aRYA'}, Request: {}, Response: HTTPSConnectionPool(host='apiconnect.angelone.in', port=443): Max retries exceeded with url: /rest/secure/angelbroking/portfolio/v1/getAllHolding?%7B%7D (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x112822fd0>: Failed to resolve 'apiconnect.angelone.in' ([Errno 8] nodename nor servname provided, or not known)"))
[E 250306 11:11:06 smartConnect:234] Error occurred while making a GET request to https://apiconnect.angelone.in/rest/secure/angelbroking/portfolio/v1/getAllHolding. Headers: {'Content-type': 'application/json', 'X-ClientLocalIP': '127.0.0.1', 'X-ClientPublicIP': '106.193.147.98', 'X-MACAddress': '6a:e0:7a:cc:09:b3', 'Accept': 'application/json', 'X-PrivateKey': '8mCgbjiE', 'X-UserType': 'USER', 'X-SourceID': 'WEB', 'Authorization': 'Bearer eyJhbGciOiJIUzUxMiJ9.eyJ1c2VybmFtZSI6IlM1NDg2NDIxNCIsInJvbGVzIjowLCJ1c2VydHlwZSI6IlVTRVIiLCJ0b2tlbiI6ImV5SmhiR2NpT2lKU1V6STFOaUlzSW5SNWNDSTZJa3BYVkNKOS5leUoxYzJWeVgzUjVjR1VpT2lKamJHbGxiblFpTENKMGIydGxibDkwZVhCbElqb2lkSEpoWkdWZllXTmpaWE56WDNSdmEyVnVJaXdpWjIxZmFXUWlPamdzSW5OdmRYSmpaU0k2SWpNaUxDSmtaWFpwWTJWZmFXUWlPaUkxWkdVM1lqazROUzB3WVRrd0xUTmtNREl0T1RZMk55MWxNVFV3TWpFNE5XVmlZamdpTENKcmFXUWlPaUowY21Ga1pWOXJaWGxmZGpJaUxDSnZiVzVsYldGdVlXZGxjbWxrSWpvNExDSndjbTlrZFdOMGN5STZleUprWlcxaGRDSTZleUp6ZEdGMGRYTWlPaUpoWTNScGRtVWlmU3dpYldZaU9uc2ljM1JoZEhWeklqb2lZV04wYVhabEluMTlMQ0pwYzNNaU9pSjBjbUZrWlY5c2IyZHBibDl6WlhKMmFXTmxJaXdpYzNWaUlqb2lVelUwT0RZME1qRTBJaXdpWlhod0lqb3hOelF4TXpJME9EQXhMQ0p1WW1ZaU9qRTNOREV5TXpneU1qRXNJbWxoZENJNk1UYzBNVEl6T0RJeU1Td2lhblJwSWpvaU1ERmhNamRsWmpBdFpqZGxaQzAwT0RCakxUazFaamt0WWpoaVpUQmlNVGt6TTJKaUlpd2lWRzlyWlc0aU9pSWlmUS53S3QtaktYVm5MMnlWeFdDSVFFYzhrTm12ZTYwbGRBU3ZCV3VuN3VQd2Y0M3Q2T2FjWmRyM0RvMDIybzVpNll2NlFXbTVZN2w0ZHh4TTVXd21GT0NyWFRBalVzY0JLUC1FOVVORWkxSS1tOUk1NU5leUgtLWtKSUJSbXpGVUZ5RG9vcEU2aHdFRHNwTk1JdUp0czJZeFFVc2plbnhTV19KNzdCbmw4UzVONTAiLCJBUEktS0VZIjoiOG1DZ2JqaUUiLCJpYXQiOjE3NDEyMzg0MDEsImV4cCI6MTc0MTMyNDgwMX0.F4eAeflNb6PzBnd02ktGJt_XVh_K6OpGlKjZgpHdgGX4lsahDfuAVJ8LZtfUnIPWmlGFVO_KBMBu8yBgY5aRYA'}, Request: {}, Response: HTTPSConnectionPool(host='apiconnect.angelone.in', port=443): Max retries exceeded with url: /rest/secure/angelbroking/portfolio/v1/getAllHolding?%7B%7D (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x112820cd0>: Failed to resolve 'apiconnect.angelone.in' ([Errno 8] nodename nor servname provided, or not known)"))
[E 250306 11:11:16 smartConnect:234] Error occurred while making a GET request to https://apiconnect.angelone.in/rest/secure/angelbroking/portfolio/v1/getAllHolding. Headers: {'Content-type': 'application/json', 'X-ClientLocalIP': '127.0.0.1', 'X-ClientPublicIP': '106.193.147.98', 'X-MACAddress': '6a:e0:7a:cc:09:b3', 'Accept': 'application/json', 'X-PrivateKey': '8mCgbjiE', 'X-UserType': 'USER', 'X-SourceID': 'WEB', 'Authorization': 'Bearer eyJhbGciOiJIUzUxMiJ9.eyJ1c2VybmFtZSI6IlM1NDg2NDIxNCIsInJvbGVzIjowLCJ1c2VydHlwZSI6IlVTRVIiLCJ0b2tlbiI6ImV5SmhiR2NpT2lKU1V6STFOaUlzSW5SNWNDSTZJa3BYVkNKOS5leUoxYzJWeVgzUjVjR1VpT2lKamJHbGxiblFpTENKMGIydGxibDkwZVhCbElqb2lkSEpoWkdWZllXTmpaWE56WDNSdmEyVnVJaXdpWjIxZmFXUWlPamdzSW5OdmRYSmpaU0k2SWpNaUxDSmtaWFpwWTJWZmFXUWlPaUkxWkdVM1lqazROUzB3WVRrd0xUTmtNREl0T1RZMk55MWxNVFV3TWpFNE5XVmlZamdpTENKcmFXUWlPaUowY21Ga1pWOXJaWGxmZGpJaUxDSnZiVzVsYldGdVlXZGxjbWxrSWpvNExDSndjbTlrZFdOMGN5STZleUprWlcxaGRDSTZleUp6ZEdGMGRYTWlPaUpoWTNScGRtVWlmU3dpYldZaU9uc2ljM1JoZEhWeklqb2lZV04wYVhabEluMTlMQ0pwYzNNaU9pSjBjbUZrWlY5c2IyZHBibDl6WlhKMmFXTmxJaXdpYzNWaUlqb2lVelUwT0RZME1qRTBJaXdpWlhod0lqb3hOelF4TXpJME9EQXhMQ0p1WW1ZaU9qRTNOREV5TXpneU1qRXNJbWxoZENJNk1UYzBNVEl6T0RJeU1Td2lhblJwSWpvaU1ERmhNamRsWmpBdFpqZGxaQzAwT0RCakxUazFaamt0WWpoaVpUQmlNVGt6TTJKaUlpd2lWRzlyWlc0aU9pSWlmUS53S3QtaktYVm5MMnlWeFdDSVFFYzhrTm12ZTYwbGRBU3ZCV3VuN3VQd2Y0M3Q2T2FjWmRyM0RvMDIybzVpNll2NlFXbTVZN2w0ZHh4TTVXd21GT0NyWFRBalVzY0JLUC1FOVVORWkxSS1tOUk1NU5leUgtLWtKSUJSbXpGVUZ5RG9vcEU2aHdFRHNwTk1JdUp0czJZeFFVc2plbnhTV19KNzdCbmw4UzVONTAiLCJBUEktS0VZIjoiOG1DZ2JqaUUiLCJpYXQiOjE3NDEyMzg0MDEsImV4cCI6MTc0MTMyNDgwMX0.F4eAeflNb6PzBnd02ktGJt_XVh_K6OpGlKjZgpHdgGX4lsahDfuAVJ8LZtfUnIPWmlGFVO_KBMBu8yBgY5aRYA'}, Request: {}, Response: HTTPSConnectionPool(host='apiconnect.angelone.in', port=443): Max retries exceeded with url: /rest/secure/angelbroking/portfolio/v1/getAllHolding?%7B%7D (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x117925090>: Failed to resolve 'apiconnect.angelone.in' ([Errno 8] nodename nor servname provided, or not known)"))
[E 250306 11:11:16 smartConnect:234] Error occurred while making a GET request to https://apiconnect.angelone.in/rest/secure/angelbroking/portfolio/v1/getAllHolding. Headers: {'Content-type': 'application/json', 'X-ClientLocalIP': '127.0.0.1', 'X-ClientPublicIP': '106.193.147.98', 'X-MACAddress': '6a:e0:7a:cc:09:b3', 'Accept': 'application/json', 'X-PrivateKey': '8mCgbjiE', 'X-UserType': 'USER', 'X-SourceID': 'WEB', 'Authorization': 'Bearer eyJhbGciOiJIUzUxMiJ9.eyJ1c2VybmFtZSI6IlM1NDg2NDIxNCIsInJvbGVzIjowLCJ1c2VydHlwZSI6IlVTRVIiLCJ0b2tlbiI6ImV5SmhiR2NpT2lKU1V6STFOaUlzSW5SNWNDSTZJa3BYVkNKOS5leUoxYzJWeVgzUjVjR1VpT2lKamJHbGxiblFpTENKMGIydGxibDkwZVhCbElqb2lkSEpoWkdWZllXTmpaWE56WDNSdmEyVnVJaXdpWjIxZmFXUWlPamdzSW5OdmRYSmpaU0k2SWpNaUxDSmtaWFpwWTJWZmFXUWlPaUkxWkdVM1lqazROUzB3WVRrd0xUTmtNREl0T1RZMk55MWxNVFV3TWpFNE5XVmlZamdpTENKcmFXUWlPaUowY21Ga1pWOXJaWGxmZGpJaUxDSnZiVzVsYldGdVlXZGxjbWxrSWpvNExDSndjbTlrZFdOMGN5STZleUprWlcxaGRDSTZleUp6ZEdGMGRYTWlPaUpoWTNScGRtVWlmU3dpYldZaU9uc2ljM1JoZEhWeklqb2lZV04wYVhabEluMTlMQ0pwYzNNaU9pSjBjbUZrWlY5c2IyZHBibDl6WlhKMmFXTmxJaXdpYzNWaUlqb2lVelUwT0RZME1qRTBJaXdpWlhod0lqb3hOelF4TXpJME9EQXhMQ0p1WW1ZaU9qRTNOREV5TXpneU1qRXNJbWxoZENJNk1UYzBNVEl6T0RJeU1Td2lhblJwSWpvaU1ERmhNamRsWmpBdFpqZGxaQzAwT0RCakxUazFaamt0WWpoaVpUQmlNVGt6TTJKaUlpd2lWRzlyWlc0aU9pSWlmUS53S3QtaktYVm5MMnlWeFdDSVFFYzhrTm12ZTYwbGRBU3ZCV3VuN3VQd2Y0M3Q2T2FjWmRyM0RvMDIybzVpNll2NlFXbTVZN2w0ZHh4TTVXd21GT0NyWFRBalVzY0JLUC1FOVVORWkxSS1tOUk1NU5leUgtLWtKSUJSbXpGVUZ5RG9vcEU2aHdFRHNwTk1JdUp0czJZeFFVc2plbnhTV19KNzdCbmw4UzVONTAiLCJBUEktS0VZIjoiOG1DZ2JqaUUiLCJpYXQiOjE3NDEyMzg0MDEsImV4cCI6MTc0MTMyNDgwMX0.F4eAeflNb6PzBnd02ktGJt_XVh_K6OpGlKjZgpHdgGX4lsahDfuAVJ8LZtfUnIPWmlGFVO_KBMBu8yBgY5aRYA'}, Request: {}, Response: HTTPSConnectionPool(host='apiconnect.angelone.in', port=443): Max retries exceeded with url: /rest/secure/angelbroking/portfolio/v1/getAllHolding?%7B%7D (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x117925590>: Failed to resolve 'apiconnect.angelone.in' ([Errno 8] nodename nor servname provided, or not known)"))
[E 250306 11:11:16 smartConnect:234] Error occurred while making a GET request to https://apiconnect.angelone.in/rest/secure/angelbroking/portfolio/v1/getAllHolding. Headers: {'Content-type': 'application/json', 'X-ClientLocalIP': '127.0.0.1', 'X-ClientPublicIP': '106.193.147.98', 'X-MACAddress': '6a:e0:7a:cc:09:b3', 'Accept': 'application/json', 'X-PrivateKey': '8mCgbjiE', 'X-UserType': 'USER', 'X-SourceID': 'WEB', 'Authorization': 'Bearer eyJhbGciOiJIUzUxMiJ9.eyJ1c2VybmFtZSI6IlM1NDg2NDIxNCIsInJvbGVzIjowLCJ1c2VydHlwZSI6IlVTRVIiLCJ0b2tlbiI6ImV5SmhiR2NpT2lKU1V6STFOaUlzSW5SNWNDSTZJa3BYVkNKOS5leUoxYzJWeVgzUjVjR1VpT2lKamJHbGxiblFpTENKMGIydGxibDkwZVhCbElqb2lkSEpoWkdWZllXTmpaWE56WDNSdmEyVnVJaXdpWjIxZmFXUWlPamdzSW5OdmRYSmpaU0k2SWpNaUxDSmtaWFpwWTJWZmFXUWlPaUkxWkdVM1lqazROUzB3WVRrd0xUTmtNREl0T1RZMk55MWxNVFV3TWpFNE5XVmlZamdpTENKcmFXUWlPaUowY21Ga1pWOXJaWGxmZGpJaUxDSnZiVzVsYldGdVlXZGxjbWxrSWpvNExDSndjbTlrZFdOMGN5STZleUprWlcxaGRDSTZleUp6ZEdGMGRYTWlPaUpoWTNScGRtVWlmU3dpYldZaU9uc2ljM1JoZEhWeklqb2lZV04wYVhabEluMTlMQ0pwYzNNaU9pSjBjbUZrWlY5c2IyZHBibDl6WlhKMmFXTmxJaXdpYzNWaUlqb2lVelUwT0RZME1qRTBJaXdpWlhod0lqb3hOelF4TXpJME9EQXhMQ0p1WW1ZaU9qRTNOREV5TXpneU1qRXNJbWxoZENJNk1UYzBNVEl6T0RJeU1Td2lhblJwSWpvaU1ERmhNamRsWmpBdFpqZGxaQzAwT0RCakxUazFaamt0WWpoaVpUQmlNVGt6TTJKaUlpd2lWRzlyWlc0aU9pSWlmUS53S3QtaktYVm5MMnlWeFdDSVFFYzhrTm12ZTYwbGRBU3ZCV3VuN3VQd2Y0M3Q2T2FjWmRyM0RvMDIybzVpNll2NlFXbTVZN2w0ZHh4TTVXd21GT0NyWFRBalVzY0JLUC1FOVVORWkxSS1tOUk1NU5leUgtLWtKSUJSbXpGVUZ5RG9vcEU2aHdFRHNwTk1JdUp0czJZeFFVc2plbnhTV19KNzdCbmw4UzVONTAiLCJBUEktS0VZIjoiOG1DZ2JqaUUiLCJpYXQiOjE3NDEyMzg0MDEsImV4cCI6MTc0MTMyNDgwMX0.F4eAeflNb6PzBnd02ktGJt_XVh_K6OpGlKjZgpHdgGX4lsahDfuAVJ8LZtfUnIPWmlGFVO_KBMBu8yBgY5aRYA'}, Request: {}, Response: HTTPSConnectionPool(host='apiconnect.angelone.in', port=443): Max retries exceeded with url: /rest/secure/angelbroking/portfolio/v1/getAllHolding?%7B%7D (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1179256d0>: Failed to resolve 'apiconnect.angelone.in' ([Errno 8] nodename nor servname provided, or not known)"))
[E 250306 11:11:49 smartConnect:234] Error occurred while making a GET request to https://apiconnect.angelone.in/rest/secure/angelbroking/portfolio/v1/getAllHolding. Headers: {'Content-type': 'application/json', 'X-ClientLocalIP': '127.0.0.1', 'X-ClientPublicIP': '106.193.147.98', 'X-MACAddress': '6a:e0:7a:cc:09:b3', 'Accept': 'application/json', 'X-PrivateKey': '8mCgbjiE', 'X-UserType': 'USER', 'X-SourceID': 'WEB', 'Authorization': 'Bearer eyJhbGciOiJIUzUxMiJ9.eyJ1c2VybmFtZSI6IlM1NDg2NDIxNCIsInJvbGVzIjowLCJ1c2VydHlwZSI6IlVTRVIiLCJ0b2tlbiI6ImV5SmhiR2NpT2lKU1V6STFOaUlzSW5SNWNDSTZJa3BYVkNKOS5leUoxYzJWeVgzUjVjR1VpT2lKamJHbGxiblFpTENKMGIydGxibDkwZVhCbElqb2lkSEpoWkdWZllXTmpaWE56WDNSdmEyVnVJaXdpWjIxZmFXUWlPamdzSW5OdmRYSmpaU0k2SWpNaUxDSmtaWFpwWTJWZmFXUWlPaUkxWkdVM1lqazROUzB3WVRrd0xUTmtNREl0T1RZMk55MWxNVFV3TWpFNE5XVmlZamdpTENKcmFXUWlPaUowY21Ga1pWOXJaWGxmZGpJaUxDSnZiVzVsYldGdVlXZGxjbWxrSWpvNExDSndjbTlrZFdOMGN5STZleUprWlcxaGRDSTZleUp6ZEdGMGRYTWlPaUpoWTNScGRtVWlmU3dpYldZaU9uc2ljM1JoZEhWeklqb2lZV04wYVhabEluMTlMQ0pwYzNNaU9pSjBjbUZrWlY5c2IyZHBibDl6WlhKMmFXTmxJaXdpYzNWaUlqb2lVelUwT0RZME1qRTBJaXdpWlhod0lqb3hOelF4TXpJME9EQXhMQ0p1WW1ZaU9qRTNOREV5TXpneU1qRXNJbWxoZENJNk1UYzBNVEl6T0RJeU1Td2lhblJwSWpvaU1ERmhNamRsWmpBdFpqZGxaQzAwT0RCakxUazFaamt0WWpoaVpUQmlNVGt6TTJKaUlpd2lWRzlyWlc0aU9pSWlmUS53S3QtaktYVm5MMnlWeFdDSVFFYzhrTm12ZTYwbGRBU3ZCV3VuN3VQd2Y0M3Q2T2FjWmRyM0RvMDIybzVpNll2NlFXbTVZN2w0ZHh4TTVXd21GT0NyWFRBalVzY0JLUC1FOVVORWkxSS1tOUk1NU5leUgtLWtKSUJSbXpGVUZ5RG9vcEU2aHdFRHNwTk1JdUp0czJZeFFVc2plbnhTV19KNzdCbmw4UzVONTAiLCJBUEktS0VZIjoiOG1DZ2JqaUUiLCJpYXQiOjE3NDEyMzg0MDEsImV4cCI6MTc0MTMyNDgwMX0.F4eAeflNb6PzBnd02ktGJt_XVh_K6OpGlKjZgpHdgGX4lsahDfuAVJ8LZtfUnIPWmlGFVO_KBMBu8yBgY5aRYA'}, Request: {}, Response: HTTPSConnectionPool(host='apiconnect.angelone.in', port=443): Max retries exceeded with url: /rest/secure/angelbroking/portfolio/v1/getAllHolding?%7B%7D (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x117926490>: Failed to resolve 'apiconnect.angelone.in' ([Errno 8] nodename nor servname provided, or not known)"))
[E 250306 11:11:59 smartConnect:234] Error occurred while making a GET request to https://apiconnect.angelone.in/rest/secure/angelbroking/portfolio/v1/getAllHolding. Headers: {'Content-type': 'application/json', 'X-ClientLocalIP': '127.0.0.1', 'X-ClientPublicIP': '106.193.147.98', 'X-MACAddress': '6a:e0:7a:cc:09:b3', 'Accept': 'application/json', 'X-PrivateKey': '8mCgbjiE', 'X-UserType': 'USER', 'X-SourceID': 'WEB', 'Authorization': 'Bearer eyJhbGciOiJIUzUxMiJ9.eyJ1c2VybmFtZSI6IlM1NDg2NDIxNCIsInJvbGVzIjowLCJ1c2VydHlwZSI6IlVTRVIiLCJ0b2tlbiI6ImV5SmhiR2NpT2lKU1V6STFOaUlzSW5SNWNDSTZJa3BYVkNKOS5leUoxYzJWeVgzUjVjR1VpT2lKamJHbGxiblFpTENKMGIydGxibDkwZVhCbElqb2lkSEpoWkdWZllXTmpaWE56WDNSdmEyVnVJaXdpWjIxZmFXUWlPamdzSW5OdmRYSmpaU0k2SWpNaUxDSmtaWFpwWTJWZmFXUWlPaUkxWkdVM1lqazROUzB3WVRrd0xUTmtNREl0T1RZMk55MWxNVFV3TWpFNE5XVmlZamdpTENKcmFXUWlPaUowY21Ga1pWOXJaWGxmZGpJaUxDSnZiVzVsYldGdVlXZGxjbWxrSWpvNExDSndjbTlrZFdOMGN5STZleUprWlcxaGRDSTZleUp6ZEdGMGRYTWlPaUpoWTNScGRtVWlmU3dpYldZaU9uc2ljM1JoZEhWeklqb2lZV04wYVhabEluMTlMQ0pwYzNNaU9pSjBjbUZrWlY5c2IyZHBibDl6WlhKMmFXTmxJaXdpYzNWaUlqb2lVelUwT0RZME1qRTBJaXdpWlhod0lqb3hOelF4TXpJME9EQXhMQ0p1WW1ZaU9qRTNOREV5TXpneU1qRXNJbWxoZENJNk1UYzBNVEl6T0RJeU1Td2lhblJwSWpvaU1ERmhNamRsWmpBdFpqZGxaQzAwT0RCakxUazFaamt0WWpoaVpUQmlNVGt6TTJKaUlpd2lWRzlyWlc0aU9pSWlmUS53S3QtaktYVm5MMnlWeFdDSVFFYzhrTm12ZTYwbGRBU3ZCV3VuN3VQd2Y0M3Q2T2FjWmRyM0RvMDIybzVpNll2NlFXbTVZN2w0ZHh4TTVXd21GT0NyWFRBalVzY0JLUC1FOVVORWkxSS1tOUk1NU5leUgtLWtKSUJSbXpGVUZ5RG9vcEU2aHdFRHNwTk1JdUp0czJZeFFVc2plbnhTV19KNzdCbmw4UzVONTAiLCJBUEktS0VZIjoiOG1DZ2JqaUUiLCJpYXQiOjE3NDEyMzg0MDEsImV4cCI6MTc0MTMyNDgwMX0.F4eAeflNb6PzBnd02ktGJt_XVh_K6OpGlKjZgpHdgGX4lsahDfuAVJ8LZtfUnIPWmlGFVO_KBMBu8yBgY5aRYA'}, Request: {}, Response: HTTPSConnectionPool(host='apiconnect.angelone.in', port=443): Max retries exceeded with url: /rest/secure/angelbroking/portfolio/v1/getAllHolding?%7B%7D (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1128234d0>: Failed to resolve 'apiconnect.angelone.in' ([Errno 8] nodename nor servname provided, or not known)"))
[E 250306 11:11:59 smartConnect:234] Error occurred while making a GET request to https://apiconnect.angelone.in/rest/secure/angelbroking/portfolio/v1/getAllHolding. Headers: {'Content-type': 'application/json', 'X-ClientLocalIP': '127.0.0.1', 'X-ClientPublicIP': '106.193.147.98', 'X-MACAddress': '6a:e0:7a:cc:09:b3', 'Accept': 'application/json', 'X-PrivateKey': '8mCgbjiE', 'X-UserType': 'USER', 'X-SourceID': 'WEB', 'Authorization': 'Bearer eyJhbGciOiJIUzUxMiJ9.eyJ1c2VybmFtZSI6IlM1NDg2NDIxNCIsInJvbGVzIjowLCJ1c2VydHlwZSI6IlVTRVIiLCJ0b2tlbiI6ImV5SmhiR2NpT2lKU1V6STFOaUlzSW5SNWNDSTZJa3BYVkNKOS5leUoxYzJWeVgzUjVjR1VpT2lKamJHbGxiblFpTENKMGIydGxibDkwZVhCbElqb2lkSEpoWkdWZllXTmpaWE56WDNSdmEyVnVJaXdpWjIxZmFXUWlPamdzSW5OdmRYSmpaU0k2SWpNaUxDSmtaWFpwWTJWZmFXUWlPaUkxWkdVM1lqazROUzB3WVRrd0xUTmtNREl0T1RZMk55MWxNVFV3TWpFNE5XVmlZamdpTENKcmFXUWlPaUowY21Ga1pWOXJaWGxmZGpJaUxDSnZiVzVsYldGdVlXZGxjbWxrSWpvNExDSndjbTlrZFdOMGN5STZleUprWlcxaGRDSTZleUp6ZEdGMGRYTWlPaUpoWTNScGRtVWlmU3dpYldZaU9uc2ljM1JoZEhWeklqb2lZV04wYVhabEluMTlMQ0pwYzNNaU9pSjBjbUZrWlY5c2IyZHBibDl6WlhKMmFXTmxJaXdpYzNWaUlqb2lVelUwT0RZME1qRTBJaXdpWlhod0lqb3hOelF4TXpJME9EQXhMQ0p1WW1ZaU9qRTNOREV5TXpneU1qRXNJbWxoZENJNk1UYzBNVEl6T0RJeU1Td2lhblJwSWpvaU1ERmhNamRsWmpBdFpqZGxaQzAwT0RCakxUazFaamt0WWpoaVpUQmlNVGt6TTJKaUlpd2lWRzlyWlc0aU9pSWlmUS53S3QtaktYVm5MMnlWeFdDSVFFYzhrTm12ZTYwbGRBU3ZCV3VuN3VQd2Y0M3Q2T2FjWmRyM0RvMDIybzVpNll2NlFXbTVZN2w0ZHh4TTVXd21GT0NyWFRBalVzY0JLUC1FOVVORWkxSS1tOUk1NU5leUgtLWtKSUJSbXpGVUZ5RG9vcEU2aHdFRHNwTk1JdUp0czJZeFFVc2plbnhTV19KNzdCbmw4UzVONTAiLCJBUEktS0VZIjoiOG1DZ2JqaUUiLCJpYXQiOjE3NDEyMzg0MDEsImV4cCI6MTc0MTMyNDgwMX0.F4eAeflNb6PzBnd02ktGJt_XVh_K6OpGlKjZgpHdgGX4lsahDfuAVJ8LZtfUnIPWmlGFVO_KBMBu8yBgY5aRYA'}, Request: {}, Response: HTTPSConnectionPool(host='apiconnect.angelone.in', port=443): Max retries exceeded with url: /rest/secure/angelbroking/portfolio/v1/getAllHolding?%7B%7D (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x112820cd0>: Failed to resolve 'apiconnect.angelone.in' ([Errno 8] nodename nor servname provided, or not known)"))
[E 250306 11:11:59 smartConnect:234] Error occurred while making a GET request to https://apiconnect.angelone.in/rest/secure/angelbroking/portfolio/v1/getAllHolding. Headers: {'Content-type': 'application/json', 'X-ClientLocalIP': '127.0.0.1', 'X-ClientPublicIP': '106.193.147.98', 'X-MACAddress': '6a:e0:7a:cc:09:b3', 'Accept': 'application/json', 'X-PrivateKey': '8mCgbjiE', 'X-UserType': 'USER', 'X-SourceID': 'WEB', 'Authorization': 'Bearer eyJhbGciOiJIUzUxMiJ9.eyJ1c2VybmFtZSI6IlM1NDg2NDIxNCIsInJvbGVzIjowLCJ1c2VydHlwZSI6IlVTRVIiLCJ0b2tlbiI6ImV5SmhiR2NpT2lKU1V6STFOaUlzSW5SNWNDSTZJa3BYVkNKOS5leUoxYzJWeVgzUjVjR1VpT2lKamJHbGxiblFpTENKMGIydGxibDkwZVhCbElqb2lkSEpoWkdWZllXTmpaWE56WDNSdmEyVnVJaXdpWjIxZmFXUWlPamdzSW5OdmRYSmpaU0k2SWpNaUxDSmtaWFpwWTJWZmFXUWlPaUkxWkdVM1lqazROUzB3WVRrd0xUTmtNREl0T1RZMk55MWxNVFV3TWpFNE5XVmlZamdpTENKcmFXUWlPaUowY21Ga1pWOXJaWGxmZGpJaUxDSnZiVzVsYldGdVlXZGxjbWxrSWpvNExDSndjbTlrZFdOMGN5STZleUprWlcxaGRDSTZleUp6ZEdGMGRYTWlPaUpoWTNScGRtVWlmU3dpYldZaU9uc2ljM1JoZEhWeklqb2lZV04wYVhabEluMTlMQ0pwYzNNaU9pSjBjbUZrWlY5c2IyZHBibDl6WlhKMmFXTmxJaXdpYzNWaUlqb2lVelUwT0RZME1qRTBJaXdpWlhod0lqb3hOelF4TXpJME9EQXhMQ0p1WW1ZaU9qRTNOREV5TXpneU1qRXNJbWxoZENJNk1UYzBNVEl6T0RJeU1Td2lhblJwSWpvaU1ERmhNamRsWmpBdFpqZGxaQzAwT0RCakxUazFaamt0WWpoaVpUQmlNVGt6TTJKaUlpd2lWRzlyWlc0aU9pSWlmUS53S3QtaktYVm5MMnlWeFdDSVFFYzhrTm12ZTYwbGRBU3ZCV3VuN3VQd2Y0M3Q2T2FjWmRyM0RvMDIybzVpNll2NlFXbTVZN2w0ZHh4TTVXd21GT0NyWFRBalVzY0JLUC1FOVVORWkxSS1tOUk1NU5leUgtLWtKSUJSbXpGVUZ5RG9vcEU2aHdFRHNwTk1JdUp0czJZeFFVc2plbnhTV19KNzdCbmw4UzVONTAiLCJBUEktS0VZIjoiOG1DZ2JqaUUiLCJpYXQiOjE3NDEyMzg0MDEsImV4cCI6MTc0MTMyNDgwMX0.F4eAeflNb6PzBnd02ktGJt_XVh_K6OpGlKjZgpHdgGX4lsahDfuAVJ8LZtfUnIPWmlGFVO_KBMBu8yBgY5aRYA'}, Request: {}, Response: HTTPSConnectionPool(host='apiconnect.angelone.in', port=443): Max retries exceeded with url: /rest/secure/angelbroking/portfolio/v1/getAllHolding?%7B%7D (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x112822fd0>: Failed to resolve 'apiconnect.angelone.in' ([Errno 8] nodename nor servname provided, or not known)"))
[E 250306 11:11:59 smartConnect:234] Error occurred while making a GET request to https://apiconnect.angelone.in/rest/secure/angelbroking/portfolio/v1/getAllHolding. Headers: {'Content-type': 'application/json', 'X-ClientLocalIP': '127.0.0.1', 'X-ClientPublicIP': '106.193.147.98', 'X-MACAddress': '6a:e0:7a:cc:09:b3', 'Accept': 'application/json', 'X-PrivateKey': '8mCgbjiE', 'X-UserType': 'USER', 'X-SourceID': 'WEB', 'Authorization': 'Bearer eyJhbGciOiJIUzUxMiJ9.eyJ1c2VybmFtZSI6IlM1NDg2NDIxNCIsInJvbGVzIjowLCJ1c2VydHlwZSI6IlVTRVIiLCJ0b2tlbiI6ImV5SmhiR2NpT2lKU1V6STFOaUlzSW5SNWNDSTZJa3BYVkNKOS5leUoxYzJWeVgzUjVjR1VpT2lKamJHbGxiblFpTENKMGIydGxibDkwZVhCbElqb2lkSEpoWkdWZllXTmpaWE56WDNSdmEyVnVJaXdpWjIxZmFXUWlPamdzSW5OdmRYSmpaU0k2SWpNaUxDSmtaWFpwWTJWZmFXUWlPaUkxWkdVM1lqazROUzB3WVRrd0xUTmtNREl0T1RZMk55MWxNVFV3TWpFNE5XVmlZamdpTENKcmFXUWlPaUowY21Ga1pWOXJaWGxmZGpJaUxDSnZiVzVsYldGdVlXZGxjbWxrSWpvNExDSndjbTlrZFdOMGN5STZleUprWlcxaGRDSTZleUp6ZEdGMGRYTWlPaUpoWTNScGRtVWlmU3dpYldZaU9uc2ljM1JoZEhWeklqb2lZV04wYVhabEluMTlMQ0pwYzNNaU9pSjBjbUZrWlY5c2IyZHBibDl6WlhKMmFXTmxJaXdpYzNWaUlqb2lVelUwT0RZME1qRTBJaXdpWlhod0lqb3hOelF4TXpJME9EQXhMQ0p1WW1ZaU9qRTNOREV5TXpneU1qRXNJbWxoZENJNk1UYzBNVEl6T0RJeU1Td2lhblJwSWpvaU1ERmhNamRsWmpBdFpqZGxaQzAwT0RCakxUazFaamt0WWpoaVpUQmlNVGt6TTJKaUlpd2lWRzlyWlc0aU9pSWlmUS53S3QtaktYVm5MMnlWeFdDSVFFYzhrTm12ZTYwbGRBU3ZCV3VuN3VQd2Y0M3Q2T2FjWmRyM0RvMDIybzVpNll2NlFXbTVZN2w0ZHh4TTVXd21GT0NyWFRBalVzY0JLUC1FOVVORWkxSS1tOUk1NU5leUgtLWtKSUJSbXpGVUZ5RG9vcEU2aHdFRHNwTk1JdUp0czJZeFFVc2plbnhTV19KNzdCbmw4UzVONTAiLCJBUEktS0VZIjoiOG1DZ2JqaUUiLCJpYXQiOjE3NDEyMzg0MDEsImV4cCI6MTc0MTMyNDgwMX0.F4eAeflNb6PzBnd02ktGJt_XVh_K6OpGlKjZgpHdgGX4lsahDfuAVJ8LZtfUnIPWmlGFVO_KBMBu8yBgY5aRYA'}, Request: {}, Response: HTTPSConnectionPool(host='apiconnect.angelone.in', port=443): Max retries exceeded with url: /rest/secure/angelbroking/portfolio/v1/getAllHolding?%7B%7D (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1062fb750>: Failed to resolve 'apiconnect.angelone.in' ([Errno 8] nodename nor servname provided, or not known)"))
[E 250306 11:11:59 smartConnect:234] Error occurred while making a GET request to https://apiconnect.angelone.in/rest/secure/angelbroking/portfolio/v1/getAllHolding. Headers: {'Content-type': 'application/json', 'X-ClientLocalIP': '127.0.0.1', 'X-ClientPublicIP': '106.193.147.98', 'X-MACAddress': '6a:e0:7a:cc:09:b3', 'Accept': 'application/json', 'X-PrivateKey': '8mCgbjiE', 'X-UserType': 'USER', 'X-SourceID': 'WEB', 'Authorization': 'Bearer eyJhbGciOiJIUzUxMiJ9.eyJ1c2VybmFtZSI6IlM1NDg2NDIxNCIsInJvbGVzIjowLCJ1c2VydHlwZSI6IlVTRVIiLCJ0b2tlbiI6ImV5SmhiR2NpT2lKU1V6STFOaUlzSW5SNWNDSTZJa3BYVkNKOS5leUoxYzJWeVgzUjVjR1VpT2lKamJHbGxiblFpTENKMGIydGxibDkwZVhCbElqb2lkSEpoWkdWZllXTmpaWE56WDNSdmEyVnVJaXdpWjIxZmFXUWlPamdzSW5OdmRYSmpaU0k2SWpNaUxDSmtaWFpwWTJWZmFXUWlPaUkxWkdVM1lqazROUzB3WVRrd0xUTmtNREl0T1RZMk55MWxNVFV3TWpFNE5XVmlZamdpTENKcmFXUWlPaUowY21Ga1pWOXJaWGxmZGpJaUxDSnZiVzVsYldGdVlXZGxjbWxrSWpvNExDSndjbTlrZFdOMGN5STZleUprWlcxaGRDSTZleUp6ZEdGMGRYTWlPaUpoWTNScGRtVWlmU3dpYldZaU9uc2ljM1JoZEhWeklqb2lZV04wYVhabEluMTlMQ0pwYzNNaU9pSjBjbUZrWlY5c2IyZHBibDl6WlhKMmFXTmxJaXdpYzNWaUlqb2lVelUwT0RZME1qRTBJaXdpWlhod0lqb3hOelF4TXpJME9EQXhMQ0p1WW1ZaU9qRTNOREV5TXpneU1qRXNJbWxoZENJNk1UYzBNVEl6T0RJeU1Td2lhblJwSWpvaU1ERmhNamRsWmpBdFpqZGxaQzAwT0RCakxUazFaamt0WWpoaVpUQmlNVGt6TTJKaUlpd2lWRzlyWlc0aU9pSWlmUS53S3QtaktYVm5MMnlWeFdDSVFFYzhrTm12ZTYwbGRBU3ZCV3VuN3VQd2Y0M3Q2T2FjWmRyM0RvMDIybzVpNll2NlFXbTVZN2w0ZHh4TTVXd21GT0NyWFRBalVzY0JLUC1FOVVORWkxSS1tOUk1NU5leUgtLWtKSUJSbXpGVUZ5RG9vcEU2aHdFRHNwTk1JdUp0czJZeFFVc2plbnhTV19KNzdCbmw4UzVONTAiLCJBUEktS0VZIjoiOG1DZ2JqaUUiLCJpYXQiOjE3NDEyMzg0MDEsImV4cCI6MTc0MTMyNDgwMX0.F4eAeflNb6PzBnd02ktGJt_XVh_K6OpGlKjZgpHdgGX4lsahDfuAVJ8LZtfUnIPWmlGFVO_KBMBu8yBgY5aRYA'}, Request: {}, Response: HTTPSConnectionPool(host='apiconnect.angelone.in', port=443): Max retries exceeded with url: /rest/secure/angelbroking/portfolio/v1/getAllHolding?%7B%7D (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x112823ed0>: Failed to resolve 'apiconnect.angelone.in' ([Errno 8] nodename nor servname provided, or not known)"))
[E 250306 11:11:59 smartConnect:234] Error occurred while making a GET request to https://apiconnect.angelone.in/rest/secure/angelbroking/portfolio/v1/getAllHolding. Headers: {'Content-type': 'application/json', 'X-ClientLocalIP': '127.0.0.1', 'X-ClientPublicIP': '106.193.147.98', 'X-MACAddress': '6a:e0:7a:cc:09:b3', 'Accept': 'application/json', 'X-PrivateKey': '8mCgbjiE', 'X-UserType': 'USER', 'X-SourceID': 'WEB', 'Authorization': 'Bearer eyJhbGciOiJIUzUxMiJ9.eyJ1c2VybmFtZSI6IlM1NDg2NDIxNCIsInJvbGVzIjowLCJ1c2VydHlwZSI6IlVTRVIiLCJ0b2tlbiI6ImV5SmhiR2NpT2lKU1V6STFOaUlzSW5SNWNDSTZJa3BYVkNKOS5leUoxYzJWeVgzUjVjR1VpT2lKamJHbGxiblFpTENKMGIydGxibDkwZVhCbElqb2lkSEpoWkdWZllXTmpaWE56WDNSdmEyVnVJaXdpWjIxZmFXUWlPamdzSW5OdmRYSmpaU0k2SWpNaUxDSmtaWFpwWTJWZmFXUWlPaUkxWkdVM1lqazROUzB3WVRrd0xUTmtNREl0T1RZMk55MWxNVFV3TWpFNE5XVmlZamdpTENKcmFXUWlPaUowY21Ga1pWOXJaWGxmZGpJaUxDSnZiVzVsYldGdVlXZGxjbWxrSWpvNExDSndjbTlrZFdOMGN5STZleUprWlcxaGRDSTZleUp6ZEdGMGRYTWlPaUpoWTNScGRtVWlmU3dpYldZaU9uc2ljM1JoZEhWeklqb2lZV04wYVhabEluMTlMQ0pwYzNNaU9pSjBjbUZrWlY5c2IyZHBibDl6WlhKMmFXTmxJaXdpYzNWaUlqb2lVelUwT0RZME1qRTBJaXdpWlhod0lqb3hOelF4TXpJME9EQXhMQ0p1WW1ZaU9qRTNOREV5TXpneU1qRXNJbWxoZENJNk1UYzBNVEl6T0RJeU1Td2lhblJwSWpvaU1ERmhNamRsWmpBdFpqZGxaQzAwT0RCakxUazFaamt0WWpoaVpUQmlNVGt6TTJKaUlpd2lWRzlyWlc0aU9pSWlmUS53S3QtaktYVm5MMnlWeFdDSVFFYzhrTm12ZTYwbGRBU3ZCV3VuN3VQd2Y0M3Q2T2FjWmRyM0RvMDIybzVpNll2NlFXbTVZN2w0ZHh4TTVXd21GT0NyWFRBalVzY0JLUC1FOVVORWkxSS1tOUk1NU5leUgtLWtKSUJSbXpGVUZ5RG9vcEU2aHdFRHNwTk1JdUp0czJZeFFVc2plbnhTV19KNzdCbmw4UzVONTAiLCJBUEktS0VZIjoiOG1DZ2JqaUUiLCJpYXQiOjE3NDEyMzg0MDEsImV4cCI6MTc0MTMyNDgwMX0.F4eAeflNb6PzBnd02ktGJt_XVh_K6OpGlKjZgpHdgGX4lsahDfuAVJ8LZtfUnIPWmlGFVO_KBMBu8yBgY5aRYA'}, Request: {}, Response: HTTPSConnectionPool(host='apiconnect.angelone.in', port=443): Max retries exceeded with url: /rest/secure/angelbroking/portfolio/v1/getAllHolding?%7B%7D (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x117926990>: Failed to resolve 'apiconnect.angelone.in' ([Errno 8] nodename nor servname provided, or not known)"))
[E 250306 11:12:09 smartConnect:234] Error occurred while making a GET request to https://apiconnect.angelone.in/rest/secure/angelbroking/portfolio/v1/getAllHolding. Headers: {'Content-type': 'application/json', 'X-ClientLocalIP': '127.0.0.1', 'X-ClientPublicIP': '106.193.147.98', 'X-MACAddress': '6a:e0:7a:cc:09:b3', 'Accept': 'application/json', 'X-PrivateKey': '8mCgbjiE', 'X-UserType': 'USER', 'X-SourceID': 'WEB', 'Authorization': 'Bearer eyJhbGciOiJIUzUxMiJ9.eyJ1c2VybmFtZSI6IlM1NDg2NDIxNCIsInJvbGVzIjowLCJ1c2VydHlwZSI6IlVTRVIiLCJ0b2tlbiI6ImV5SmhiR2NpT2lKU1V6STFOaUlzSW5SNWNDSTZJa3BYVkNKOS5leUoxYzJWeVgzUjVjR1VpT2lKamJHbGxiblFpTENKMGIydGxibDkwZVhCbElqb2lkSEpoWkdWZllXTmpaWE56WDNSdmEyVnVJaXdpWjIxZmFXUWlPamdzSW5OdmRYSmpaU0k2SWpNaUxDSmtaWFpwWTJWZmFXUWlPaUkxWkdVM1lqazROUzB3WVRrd0xUTmtNREl0T1RZMk55MWxNVFV3TWpFNE5XVmlZamdpTENKcmFXUWlPaUowY21Ga1pWOXJaWGxmZGpJaUxDSnZiVzVsYldGdVlXZGxjbWxrSWpvNExDSndjbTlrZFdOMGN5STZleUprWlcxaGRDSTZleUp6ZEdGMGRYTWlPaUpoWTNScGRtVWlmU3dpYldZaU9uc2ljM1JoZEhWeklqb2lZV04wYVhabEluMTlMQ0pwYzNNaU9pSjBjbUZrWlY5c2IyZHBibDl6WlhKMmFXTmxJaXdpYzNWaUlqb2lVelUwT0RZME1qRTBJaXdpWlhod0lqb3hOelF4TXpJME9EQXhMQ0p1WW1ZaU9qRTNOREV5TXpneU1qRXNJbWxoZENJNk1UYzBNVEl6T0RJeU1Td2lhblJwSWpvaU1ERmhNamRsWmpBdFpqZGxaQzAwT0RCakxUazFaamt0WWpoaVpUQmlNVGt6TTJKaUlpd2lWRzlyWlc0aU9pSWlmUS53S3QtaktYVm5MMnlWeFdDSVFFYzhrTm12ZTYwbGRBU3ZCV3VuN3VQd2Y0M3Q2T2FjWmRyM0RvMDIybzVpNll2NlFXbTVZN2w0ZHh4TTVXd21GT0NyWFRBalVzY0JLUC1FOVVORWkxSS1tOUk1NU5leUgtLWtKSUJSbXpGVUZ5RG9vcEU2aHdFRHNwTk1JdUp0czJZeFFVc2plbnhTV19KNzdCbmw4UzVONTAiLCJBUEktS0VZIjoiOG1DZ2JqaUUiLCJpYXQiOjE3NDEyMzg0MDEsImV4cCI6MTc0MTMyNDgwMX0.F4eAeflNb6PzBnd02ktGJt_XVh_K6OpGlKjZgpHdgGX4lsahDfuAVJ8LZtfUnIPWmlGFVO_KBMBu8yBgY5aRYA'}, Request: {}, Response: HTTPSConnectionPool(host='apiconnect.angelone.in', port=443): Max retries exceeded with url: /rest/secure/angelbroking/portfolio/v1/getAllHolding?%7B%7D (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x117924f50>: Failed to resolve 'apiconnect.angelone.in' ([Errno 8] nodename nor servname provided, or not known)"))
[E 250306 11:12:09 smartConnect:234] Error occurred while making a GET request to https://apiconnect.angelone.in/rest/secure/angelbroking/portfolio/v1/getAllHolding. Headers: {'Content-type': 'application/json', 'X-ClientLocalIP': '127.0.0.1', 'X-ClientPublicIP': '106.193.147.98', 'X-MACAddress': '6a:e0:7a:cc:09:b3', 'Accept': 'application/json', 'X-PrivateKey': '8mCgbjiE', 'X-UserType': 'USER', 'X-SourceID': 'WEB', 'Authorization': 'Bearer eyJhbGciOiJIUzUxMiJ9.eyJ1c2VybmFtZSI6IlM1NDg2NDIxNCIsInJvbGVzIjowLCJ1c2VydHlwZSI6IlVTRVIiLCJ0b2tlbiI6ImV5SmhiR2NpT2lKU1V6STFOaUlzSW5SNWNDSTZJa3BYVkNKOS5leUoxYzJWeVgzUjVjR1VpT2lKamJHbGxiblFpTENKMGIydGxibDkwZVhCbElqb2lkSEpoWkdWZllXTmpaWE56WDNSdmEyVnVJaXdpWjIxZmFXUWlPamdzSW5OdmRYSmpaU0k2SWpNaUxDSmtaWFpwWTJWZmFXUWlPaUkxWkdVM1lqazROUzB3WVRrd0xUTmtNREl0T1RZMk55MWxNVFV3TWpFNE5XVmlZamdpTENKcmFXUWlPaUowY21Ga1pWOXJaWGxmZGpJaUxDSnZiVzVsYldGdVlXZGxjbWxrSWpvNExDSndjbTlrZFdOMGN5STZleUprWlcxaGRDSTZleUp6ZEdGMGRYTWlPaUpoWTNScGRtVWlmU3dpYldZaU9uc2ljM1JoZEhWeklqb2lZV04wYVhabEluMTlMQ0pwYzNNaU9pSjBjbUZrWlY5c2IyZHBibDl6WlhKMmFXTmxJaXdpYzNWaUlqb2lVelUwT0RZME1qRTBJaXdpWlhod0lqb3hOelF4TXpJME9EQXhMQ0p1WW1ZaU9qRTNOREV5TXpneU1qRXNJbWxoZENJNk1UYzBNVEl6T0RJeU1Td2lhblJwSWpvaU1ERmhNamRsWmpBdFpqZGxaQzAwT0RCakxUazFaamt0WWpoaVpUQmlNVGt6TTJKaUlpd2lWRzlyWlc0aU9pSWlmUS53S3QtaktYVm5MMnlWeFdDSVFFYzhrTm12ZTYwbGRBU3ZCV3VuN3VQd2Y0M3Q2T2FjWmRyM0RvMDIybzVpNll2NlFXbTVZN2w0ZHh4TTVXd21GT0NyWFRBalVzY0JLUC1FOVVORWkxSS1tOUk1NU5leUgtLWtKSUJSbXpGVUZ5RG9vcEU2aHdFRHNwTk1JdUp0czJZeFFVc2plbnhTV19KNzdCbmw4UzVONTAiLCJBUEktS0VZIjoiOG1DZ2JqaUUiLCJpYXQiOjE3NDEyMzg0MDEsImV4cCI6MTc0MTMyNDgwMX0.F4eAeflNb6PzBnd02ktGJt_XVh_K6OpGlKjZgpHdgGX4lsahDfuAVJ8LZtfUnIPWmlGFVO_KBMBu8yBgY5aRYA'}, Request: {}, Response: HTTPSConnectionPool(host='apiconnect.angelone.in', port=443): Max retries exceeded with url: /rest/secure/angelbroking/portfolio/v1/getAllHolding?%7B%7D (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x117924e10>: Failed to resolve 'apiconnect.angelone.in' ([Errno 8] nodename nor servname provided, or not known)"))
